### PR TITLE
React app searching

### PR DESCRIPTION
**Pull request by ajs476**

### CODE REVIEW

File: posters-project/src/App.js [hunk 1]
```diff
@@ -27,20 +29,37 @@ class App extends React.Component {
   }
 
   searchOMDB(searchTitle) {
-    qhttp.read(`http://www.omdbapi.com/?apikey=fd86ad97&t=${searchTitle}`)
+    if (searchTitle == '') {
```
Comment (lucas-aragno) [inline posters-project/src/App.js]: use always 3 `=`

**Review by lucas-aragno — CHANGES_REQUESTED** [review]

File: posters-project/src/App.js (hunk 1 above)
Comment (ajs476) [reply posters-project/src/App.js]: I see. Done!

File: posters-project/src/App.js [hunk 2]
```diff
@@ -27,20 +29,37 @@ class App extends React.Component {
...
-      this.updateMovies({title: omdbResult.Title, genre: omdbResult.Genre, posterImageURL: omdbResult.Poster, userImageURL: omdbResult.Poster});
+      if (omdbResult.Response === 'False') {
+        this.setState({statusMessage: `Error: ${omdbResult.Error}`});
+        return;
+      }
+      let formattedResults = omdbResult.Search.map(({Title: title, Year: genre, Poster: posterImageURL}) =>
+        ({title, genre, posterImageURL, userImageURL: posterImageURL})
+      );
+      this.updateMovies(formattedResults);
+      this.setState({statusMessage: ''});
```
Comment (lucas-aragno) [inline posters-project/src/App.js]: can you try `this.setState({statusMesage: ''}, () => this.updateMovies(formattedResults))`

**Review by lucas-aragno — CHANGES_REQUESTED** [review]

File: posters-project/src/App.js (hunk 2 above)
Comment (ajs476) [reply posters-project/src/App.js]: This basically ensures that the status is set before the movies get updated right?

Comment (ajs476) [reply posters-project/src/App.js]: Done.

**Review by lucas-aragno — APPROVED** [review]

**Pull request merged**